index.bs

```diff
@@ -36,564 +36,122 @@ THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-AVIF file. It is typically of a smaller scale than the Master Images. Its compression format may be
-different than the one used by the Master Images.
-</p>
+<p>An AVIF file containing an [=AV1 Image Sequence=] SHALL list the <code>msf1</code> brand as one of the entries in the <code>compatible_brands</code> array and conform to sections 7 and 10.3 of [[!HEIF]].</p-->
 
-<h2 id="object-model-and-structure">Object Model and Structure</h2>
+<h2 id="image-item-and-properties">Image Items and properties</h2>
 
-<p>An AVIF file shall be a conformant version of an [[!HEIF]] file.
-This is to allow for the deployment of general libraries that may be used to create and parse
-HEIF-based image files wrapping different coding methods for the actual image content.
-This should be similar to ISO-BMFF usage in the video domain.
-</p>
-<p>
-The AVIF file format will be built on the box-structured media interchange format introduced
-by the ISO Base Media File Format ([[!ISOBMFF]]). The format specified by AVIF defines the use
-of a subset of box structures introduced in ISOBMFF. Where the necessary structures do not
-exist in ISOBMFF, structures defined as part of the High Efficiency Image File Format
-([[!HEIF]]: ISO/IEC 23008-12) that are codec neutral and can be applied in a generic manner
-are used. An AVIF version 1.0 file shall be compliant to the requirements of Clause 4 of the
-[[!ISOBMFF]] specification, and where applicable, the recommendations in Annex I: Guidelines
-On Defining New Image Formats and Brands in the MPEG HEIF specification shall be followed
-for AVIF 1.0.
-</p>
+<h3 id="image-item">AV1 Image Item</h3>
 
-<h2 id="avif-image-data">Image Data</h2>
-
-<p> Image data of type "av1i" shall be limited to AV1 intra frames. This applies to both Image Collections and Image Sequences.
-Each image shall conform to the requirements of an Intra Frame as defined by AV1 Bitstream & Decoding Process Specification
-[[!AV1-ISOBMFF]]. No inter-frame encoding shall be permitted between images.
-</p>
-
-<p>
-The image data shall have the "still_picture" field set to 1 in the sequence header OBU as specified in section 5.5.1 of the Bitstream & Decoding Process Specification [[!AV1]].
-</p>
-
-<p>
-The image data should have the "reduced_still_picture_header" field set to 1 in the sequence header OBU as specified in section 5.5.1 of the Bitstream & Decoding Process Specification [[!AV1]]. The field may be set to 0 to allow for easy extraction of frames from an AV1 file.
-</p>
-
-<h2 id="avif=image-collection">AVIF Image Collection</h2>
-
-<p>The image data of type "av1i" shall be used for an image collection item coded with AV1.
-
-The image item data shall be structured as defined in the AV1 Sample Format section of the AV1 Codec ISO Media File Format Binding
-[[!AV1]] specification.
+<p>When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and SHALL obey the following constraints:
```

Comment:
Can you share the motivation behind the naming to `av01` compared to `av1i`?

Reply:
Definitely a discussion we could have. See also #9.

Annex J of HEIF says:
> It is suggested that image sequence tracks are specified like video tracks of the same coding format with a change of handler type (from 'vide' to 'pict') and possibly with additional constraints.

and
>In coded image items, the coding type is identified by the item_type, and in image sequences, by the sample entry type. It is suggested that the same code be used for the two cases;

I just followed these suggestions.